includes/Shopping/Woocommerce_Query.php

```diff
@@ -66,19 +66,23 @@ public function get_search( string $search_term ) {
 
 		$product_image_ids = [];
 		foreach ( $products as $product ) {
-			$product_image_ids[] = $product->get_gallery_image_ids();
+			$product_image_ids[] = $this->get_product_image_ids( $product );
 		}
 		$products_image_ids = array_merge( [], ...$product_image_ids );
+
 		/**
 		 * Warm the object cache with post and meta information for all found
 		 * images to avoid making individual database calls.
 		 */
 		_prime_post_caches( $products_image_ids, false, true );
 
 		foreach ( $products as $product ) {
-			$product_image_ids = array_map( 'absint', $product->get_gallery_image_ids() );
-			$images            = array_map( [ $this, 'get_product_images' ], $product_image_ids );
-
+			
+			$images = array_map(
+				[ $this, 'get_product_image' ],
+				$this->get_product_image_ids( $product )
+			);
```

Comment:
@swissspidy What size image should we be loading here?

Reply:
Great question, only just saw it now though.

Using `large` should suffice.